include/ignition/gazebo/ServerConfig.hh

```diff
@@ -262,6 +262,19 @@ namespace ignition
       /// \TODO(anyone) Deprecate on Dome, SDF path will always be ignored.
       public: void SetLogIgnoreSdfPath(bool _ignore);
 
+      /// \brief Add a topic to record.
+      /// \param[in] _topic Topic name, which can include wildcards.
+      public: void AddLogRecordTopic(const std::string _topic);
+
+      /// \brief Clear topics to record. This will remove all topics set
+      /// using AddLogRecordTopic.
+      public: void ClearLogRecordTopics();
+
+      /// \brief Get the topics to record that were added using
+      /// AddLogRecordTopic.
+      /// \return The topics to record.
+      public: const std::vector<std::string> &LogRecordTopics() const;
```

Comment:
cppcheck

`./include/ignition/gazebo/ServerConfig.hh:276:  Add #include <vector> for vector<>  [build/include_what_you_use] [4]`